edx_lint/cmd/amnesty.py

```diff
@@ -0,0 +1,110 @@
+"""
+Commandline scripts to give one-time amnesty for specify linting error conditions.
+"""
+
+from __future__ import print_function
+
+from collections import namedtuple, defaultdict
+import re
+import sys
+
+import click
+
+PYLINT_PARSEABLE_REGEX = re.compile(
+    r"""^(?P<filename>[^:]+):(?P<line>\d+): """
+    r"""\[(?P<error_number>[^(]+)\((?P<error_code>[^)]+)\), (?P<function>[^\]]*)\] """
+    r"""(?P<error_msg>.*)"""
+)
+PYLINT_EXCEPTION_REGEX = re.compile(r"""\s*#\s*pylint:\s+disable=(?P<disables>[^#]*)""")
+
+PylintError = namedtuple('PylintError', ['filename', 'line', 'error_number', 'error_code', 'function', 'error_msg'])
```

Comment:
Here "line" means line number, later in fix_pylint, it means the text of the line. "linenum" here would be good.